pkgs/development/tools/wasm-pack/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, rustPlatform, pkgconfig, openssl }:
+
+rustPlatform.buildRustPackage rec {
+  name = "wasm-pack-${version}";
+  version = "0.8.1";
+
+  src = fetchFromGitHub {
+    owner = "rustwasm";
+    repo = "wasm-pack";
+    rev = "v${version}";
+    sha256 = "1z66m16n4r16zqmnv84a5jndr5x6mdqdq4b1wq929sablwqd2rl4";
+  };
+
+  cargoSha256 = "1xdx0gjqd4zyhnp72hz88rdmgry1m7rcw2j73lh67vp08z74y54y";
+
+  nativeBuildInputs = [ pkgconfig openssl ];
```

Comment:
This probably links against openssl, in which case it should be a `buildInput`.

Reply:
@LnL7 yes indeed!

For cross-compilation, is this host/build distinction more significant than just for clarity?

Reply:
@LnL7 After re-reading the relevant sections of nixpkgs manual and observing the build result on darwin, I think I now understand the distinction between `nativeBuildInputs` and `buildInputs`.

In my latest change, I've put `pkgconfig` into `naitveBuildInputs` and `openssl` into `buildInputs`:

```
nativeBuildInputs = [ pkgconfig ];

buildInputs = [ openssl ];
```

Thanks again for reviewing this PR and suggesting changes.

Is there more work to be done to get this PR fit for merge?